docs/how-to/authentication-plugin-examples.md

```diff
@@ -245,3 +249,70 @@ The Keycloak Auth Provider integrates Keycloak authentication and authorization
 - `recipe`: Configuration for the authentication recipe
 
 For detailed setup instructions and configuration options, refer to the [Keycloak Auth Provider GitHub README](https://github.com/amplication/plugins/tree/master/plugins/auth-keycloak).
+
+
+## Real Examples
+
+This section provides you with in-depth examples on how to use and interact with some of the available authentication plugins.
+
+### JWT Authentication (Node.js)
+
+When generating an app with JWT authentication, the process includes the following two steps:
+
+1. Send a login request to the server with username and password to get back from the server the JWT token.
+2. Add an authentication header with the JWT token to every consecutive request.
+
+Following are examples of how to log in with REST API and GraphQL API.
+
+#### Rest API
+
+```bash title="Rest API Curl Request Example"
+curl -X 'POST' \
+  'https://[server-url]/api/login' \
+  -H 'accept: */*' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "username": "admin",
+  "password": "admin"
+}'
+```
+
+![Swagger playground for login endpoint.](./../getting-started/assets/authentication/auth-rest.png)
+
+#### GraphQL API
+
+```graphql
+mutation {
+  login(credentials: { username: "admin", password: "admin" }) {
+    accessToken
+  }
+}
+```
+
+#### Header with JWT Included (example)
+
+<!-- spell-checker: disable -->
+
+```text title="JWT Authorization Header Example"
+ Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjoieW91IGFyZSBzb29vb28gY29vbCB0aGF0IHlvdSBjaGVjayB0aGF0ISIsIm5hbWUiOiJPZmVrIGdhYmF5IDspIiwiaWF0IjoxNTE2MjM5MDIyfQ.vaYJaP9SUlOU0u4NfFCRm5tmBVDKeCwvN6ByCkqJt8U
+```
+
+<!-- spell-checker: enable -->
+
+### Basic Authentication (Node.js)
+
+When using Basic HTTP authentication, when sending a request to the API you must provide a Basic HTTP authentication header with the format:
+
+Authorization: 'type' 'credentials'
+
+where type is Basic and credentials is the Base64 encoding of a string "username:password".
+
+```text title="Basic Authentication Example"
+Authorization: Basic YWRtaW46YWRtaW4=
+```
+
+By default, your app comes with one user with the username `admin` and password `admin`.
```

Comment:
Is it relevant to the example?
I think this info is important and we should place it much higher in the doc and in a more dominant way